python-backend/app/api/permits/permit_amendment/resources/permit_amendment_document.py

```diff
@@ -0,0 +1,125 @@
+import decimal
+import uuid
+import base64
+import requests
+import json
+
+from datetime import datetime
+from flask import request, current_app, Response
+from flask_restplus import Resource, reqparse
+from werkzeug.datastructures import FileStorage
+from werkzeug import exceptions
+from sqlalchemy.exc import DBAPIError
+
+from ..models.permit_amendment import PermitAmendment
+from ..models.permit_amendment_document import PermitAmendmentDocument
+
+from app.extensions import api, db
+from ....utils.access_decorators import requires_role_mine_create
+from ....utils.resources_mixins import UserMixin, ErrorMixin
+from ....utils.url import get_document_manager_svc_url
+
+
+class PermitAmendmentDocumentResource(Resource, UserMixin, ErrorMixin):
+    parser = reqparse.RequestParser()
+    parser.add_argument('document_manager_guid', type=str)
+    parser.add_argument('filename', type=str)
+    #permit_guid, it could be in the request and we don't want to disallow it, but it is not used.
```

Comment:
Is this kept around for a reason or just missed?

Reply:
It explains what appears to be an unnecessary parameter in all the methods in the file. I'm not attached if you think the comment is out of place.

Reply:
I'm fine with it. Just wanted to confirm that it wasn't left accidentally